proto/viam/buf.yaml

```diff
@@ -18,6 +18,5 @@ lint:
       - component/camera/v1/camera.proto
       - component/audioinput/v1/audioinput.proto
     RPC_REQUEST_RESPONSE_UNIQUE:
-      - component/camera/v1/camera.proto
-      - component/audioinput/v1/audioinput.proto
+      - component
```

Comment:
I've ignored (for all of the `component` directory) the default [`buf lint` requirement](https://docs.buf.build/lint/configuration#rpc_allow_) that all rpc methods have unique request and response message types since it defeats the point of this project. We'll have to ignore `service` as well, @purplenicole730.

Reply:
How do we ignore the requirements while running `make dist/buf`?

Reply:
As discussed offline you'll have to add `- service` here, too. That should remove the warning for both linting and compilation.